### ISSUE TITLE
"Use Ethereum Privately" Campaign

### ISSUE THREAD

**Issue by burningtree** [web3privacy/projects#11]
## Concept
An awareness and education campaign aimed at Ethereum (EVM) users to explain the disadvantages of Ethereum from a privacy perspective, and to offer methods for users to defend themselves.

## Key components
- Interactive web presentation (micro-site)
- Campaign on social networks
- Promotional merch

**Comment by Msiusko** [comment]
This could go well after we publish Ethereum Privacy Ecosystem [report](https://docs.web3privacy.info/research/ethereum-privacy-ecosystem)

Specifically, because

**Key data**. We will collect the necessary data to use in decks, on the website, socials etc

**Resources**. Advocacy is a time-consuming effort, especially because it needs to be done regularly (like 2-3 content pieces per week, 1 podcast/Twitter spaces, 1 micro-Twitter guide or similar).

**Comment by burningtree** [comment]
@Msiusko Exactly, web3privacy/research#8 can complement this campaign perfectly.

Regarding resources - as you say, it will be difficult, but if we can distribute the workload of the campaign to both our members and other partners, it might not be a problem. Twitter spaces and podcasts can always be hosted by someone else, as can posts, lots of retweets etc. 

If we want a campaign like this to have a hit, we'll need a lot of support from the community. So the biggest job will not be to make a website, prepare materials, or run Twitter. But to find enough active supporters. We can only do this together as "collective advocacy".

**Comment by Msiusko** [comment]
Such a big project like this should be re-calibrated into smaller & practical initiatives. To match the gap between the vision & execution with scarce resources.

For example, if W3PN have 3-4 active people who want to do things now -> check their skillset & adjust micro-activation no the team capabilities (example: _1 designer + 1 dev_).

Ah, I forgot: I won't attack Ethereum (disadvantages), but position it as a rode to better privacy:
- privacy on Ethereum demistified 
- what person can do now
- what to expect from the future
- how can you empower Ethereum privacy

Should be on the positive side.
